public/app/features/alerting/unified/api/alertRuleApi.ts

```diff
@@ -68,6 +71,13 @@ export interface Rule {
 
 export type AlertInstances = Record<string, string>;
 
+export interface ModifyExportPayload {
+  rules: Array<RulerAlertingRuleDTO | RulerRecordingRuleDTO | PostableRuleGrafanaRuleDTO>;
```

Comment:
Do we intend to support all three types of rules here? Isn't `PostableRuleGrafanaRuleDTO` enough?

Reply:
you're right